KEEP/services/dropbear

```diff
@@ -7,11 +7,17 @@ make_key_if_needed() {
 	if [ ! -f "$filename" ] ; then
 		local bits=1024
 		[ "$key" = "rsa" ] && bits=2048
-		dropbearkey -s $bits -t $key -f "$filename" | tail -n2 > ${filename}.pub
+		[ "$key" = "ecdsa" ] && bits=384
+		[ "$key" = "ed25519" ] && unset bits
+
+		[ ! "$key" = "ed25519" ] && dropbearkey -s $bits -t $key -f "$filename" | tail -n2 > ${filename}.pub
```

Comment:
can you turn this into an if / else ? or just turn the bits parameter into bits="-s 2048" etc and bits="" for ed5519